app/src/main/kotlin/com/github/gotify/init/InitializationActivity.kt

```diff
@@ -85,6 +85,8 @@ internal class InitializationActivity : AppCompatActivity() {
         if (manager?.canScheduleExactAlarms() == true) {
             tryAuthenticate()
         } else {
+            splashScreenActive = false
+            setContentView(R.layout.splash)
```

Comment:
Use stopSplashScreen() here too.

Reply:
True, even found another one to replace.